force.go

```diff
@@ -56,6 +57,22 @@ type QueryResult struct {
 	Records        []SObject `json:"records"`
 }
 
+// Expose sid to save in admin settings
+func (client *Client) GetSid() (sid string) {
+        return client.sessionID
+}
+
+//Expose Loc to save in admin settings
+func (client *Client) GetLoc() (loc string) {
+	return client.instanceURL
+}
+
+// Set SID and Loc as a means to log in without LoginPassword
+func (client *Client) SetSidLoc(sid string, loc string) {
+        client.sessionID = sid
+        client.instanceURL = loc
+}
+
```

Comment:
These changes are to avoid spamming LoginPassword if we already have credentials to log in.